SoloProjects/clc80/Dashboard/Dashboard/DashboardCard.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  DashboardCard.swift
+//  Dashboard
+//
+//  Created by Claudia Maciel on 12/13/20.
+//
+
+import UIKit
+
+class DashboardCard: UIView {
+    //MARK: - Properties
+    var image1 = UIImageView()
+    var image2 = UIImageView()
+    let title = UILabel()
+    let subTitle = UILabel()
```

Comment:
I recommend making these `private` unless there's a good reason for another object to be able to reach in and access these things directly.

```suggestion
    private var image1 = UIImageView()
    private var image2 = UIImageView()
    private let title = UILabel()
    private let subTitle = UILabel()
```